### PR TITLE
portals: get provisioning template runner back

### PR DESCRIPTION
**Pull request by mmadariaga**
Fixed path and minor improvements

### CODE REVIEW

File: portals/application/bin/template.php [hunk 1]
```diff
@@ -12,12 +12,11 @@
 // Define path to application directory
 $route = '';
 if ( APPLICATION_ENV != 'localdev'){
```
Comment (Kaian) [inline portals/application/bin/template.php]: I think this environment no longer exists. We only use development and production.

File: portals/application/bin/template.php [hunk 2]
```diff
@@ -47,25 +46,25 @@
 
 $scriptOptions = $application->getOption('import');
 
-$miClase = new miclase($argv[1]);
+$miClase = new TemplateWrapper($argv[1]);
```
Comment (Kaian) [inline portals/application/bin/template.php]: We could also go a bit further and also name the variable $wrapper :innocent:

File: portals/application/controllers/KlearCustomRunCodeController.php [hunk 3]
```diff
@@ -77,8 +78,7 @@ protected function _runCode($type, $inputMac, $error = false){
                 $this->view->terminalModel = $terminalModelsModel;
 
                 $var = base64_encode(serialize($this->view->getVars()));
-                $command = "/usr/bin/php ".$route . DIRECTORY_SEPARATOR .$filename." ".$var." 2>&1"; 
-
+                $command = "/usr/bin/php ".$route . DIRECTORY_SEPARATOR .$filename." ".$var." 2>&1";
```
Comment (Kaian) [inline portals/application/controllers/KlearCustomRunCodeController.php]: Not sure what changed in this line :thinking: 
The contatenation spaces are random, as it seems everywhere in this code.
But don't worry, we'll fix that in a future (never).

File: portals/application/bin/template.php (hunk 2 above)
Comment (mmadariaga) [inline portals/application/bin/template.php]: Sure :)

**Pull request merged**